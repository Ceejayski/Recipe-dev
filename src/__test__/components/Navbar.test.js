import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import NavBar from '../../components/Navbar';

test('renders Correctly', () => {
  const Placeholder = () => (
    <BrowserRouter>
      <NavBar />
    </BrowserRouter>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
