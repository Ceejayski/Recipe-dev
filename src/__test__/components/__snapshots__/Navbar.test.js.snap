// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Correctly 1`] = `
<header>
  <nav
    className="transparent navbar navbar-expand-md navbar-light fixed-top"
  >
    <div
      className="container"
    >
      <a
        className="brand navbar-brand"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="logo"
          className="d-inline-block align-top"
          height="30"
          src="logo.png"
          width="30"
        />
        RecipeDev
      </a>
      <button
        aria-controls="basic-navbar-nav"
        aria-label="Toggle navigation"
        className="navbar-toggler collapsed"
        onClick={[Function]}
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        aria-expanded={null}
        className="navbar-collapse collapse"
        id="basic-navbar-nav"
      >
        <div
          className="mx-auto w-50"
        >
          <div
            className="w-100 search-bar"
          >
            <div
              className=" css-b62m3t-container"
              onKeyDown={[Function]}
            >
              <span
                className="css-1f43avz-a11yText-A11yText"
                id="react-select-2-live-region"
              />
              <span
                aria-atomic="false"
                aria-live="polite"
                aria-relevant="additions text"
                className="css-1f43avz-a11yText-A11yText"
              />
              <div
                className=" css-192b12i-control"
                onMouseDown={[Function]}
                onTouchEnd={[Function]}
              >
                <div
                  className=" css-319lph-ValueContainer"
                >
                  <div
                    className=" css-14el2xx-placeholder"
                    id="react-select-2-placeholder"
                  >
                    Search for your Recipe here
                  </div>
                  <div
                    className=" css-6j8wv5-Input"
                    data-value=""
                  >
                    <input
                      aria-autocomplete="list"
                      aria-controls="react-select-2-listbox"
                      aria-describedby="react-select-2-placeholder"
                      aria-expanded={false}
                      aria-haspopup={true}
                      aria-owns="react-select-2-listbox"
                      autoCapitalize="none"
                      autoComplete="off"
                      autoCorrect="off"
                      className=""
                      disabled={false}
                      id="react-select-2-input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      role="combobox"
                      spellCheck="false"
                      style={
                        Object {
                          "background": 0,
                          "border": 0,
                          "color": "inherit",
                          "font": "inherit",
                          "gridArea": "1 / 2",
                          "label": "input",
                          "margin": 0,
                          "minWidth": "2px",
                          "opacity": 1,
                          "outline": 0,
                          "padding": 0,
                          "width": "100%",
                        }
                      }
                      tabIndex={0}
                      type="text"
                      value=""
                    />
                  </div>
                </div>
                <div
                  className=" css-1hb7zxy-IndicatorsContainer"
                >
                  <span
                    className=" css-1okebmr-indicatorSeparator"
                  />
                  <div
                    aria-hidden="true"
                    className=" css-tlfecz-indicatorContainer"
                    onMouseDown={[Function]}
                    onTouchEnd={[Function]}
                  >
                    <i
                      className="fas fa-search"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="navbar-nav"
          onKeyDown={[Function]}
        >
          <a
            className="nav-link"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </div>
        <div
          className="nav-item dropdown"
        >
          <a
            aria-expanded={false}
            className="dropdown-toggle nav-link"
            id="basic-nav-dropdown"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            Recipe Categories
          </a>
        </div>
      </div>
    </div>
  </nav>
</header>
`;
